core/src/components/modal/test/basic/modal.e2e.ts

```diff
@@ -3,40 +3,36 @@ import { test, Viewports } from '@utils/test/playwright';
 import type { E2EPage } from '@utils/test/playwright';
 
 test.describe('modal: focus trapping', () => {
+  test.beforeEach(async ({ browserName }, testInfo) => {
+    test.skip(testInfo.project.metadata.rtl === true, 'This does not test LTR vs. RTL layout.');
+    test.skip(browserName === 'firefox', 'Firefox incorrectly allows keyboard focus to move to ion-content');
```

Comment:
This was flaky before, but now that we have the `await` in place it was failing more often. This appears to be a Firefox bug